tests/closure/texplicit_dummy_closure.nim

```diff
@@ -1,3 +1,6 @@
+discard """
+  disabled: true
+"""
 
 # This is a regression of the new lambda lifting; detected by Aporia
```

Comment:
Oh nooo. This means Aporia won't compile at all now. IIRC it still uses asyncio D:

Not something I expect you to fix, just makes me sad :)

Reply:
Maybe update Aporia then?